README.md

```diff
@@ -0,0 +1,69 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+ portfolio
+
+> Building mobile skeleton.
+
+
+## Built With
+
+- Major languages(html and css)
+- Frameworks
+- Technologies used( linters)
+
+## Live Demo (if available)
+
+[Live Demo Link](https:jemutakilagat.github.io)
+
+
```

Comment:
- [ ] Great job so far, the link for the live demo isn't working properly, kindly fix that. after the protocol `https` it should be `://`.
- [ ] To make your readme professional please customize it to your project.